Sources/Apodini/Apodini.docc/Extensions/Parameter.md

```diff
@@ -0,0 +1,398 @@
+# ``Apodini/Parameter``
```

Comment:
I would move this in the same group as the other property wrappers. Should me merged with the other `Parameter.md`